### PR TITLE
Asp mods

### PR DESCRIPTION
**Pull request by andyparsons**
learning a little Hugo and minor changes made

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -7,5 +7,4 @@ The Coalition for Content Provenance and Authenticity (C2PA) addresses the preva
 
 C2PA unifies the efforts of the Adobe-led [Content Authenticity Initiative (CAI)](https://contentauthenticity.org/) which focuses on systems to provide context and history for digital media, and [Project Origin](https://www.originproject.info/), a Microsoft- and BBC-led initiative that tackles disinformation in the digital news ecosystem.
 
-[Graphic with paying member companies] 
-[Graphic with CAI and Project Origin Logos]
+
```
Comment (lrosenthol) [inline content/en/_index.md:10]: Was actually leaving those in as I am going to put the logos in tomorrow!

File: config.toml [hunk 2]
```diff
@@ -32,7 +32,7 @@ pluralizelisttitles = false
   twitter = ""
   instagram = ""
   youtube = ""
-  github = "https://github.com/ccpa-org"
+  github = "https://github.com/c2pa-org"
```
Comment (lrosenthol) [inline config.toml:35]: Thanks!

**Review by lrosenthol — CHANGES_REQUESTED** [review]

File: content/en/_index.md (hunk 1 above)
Comment (andyparsons) [reply content/en/_index.md:10]: 👍

File: config.toml (hunk 2 above)
Comment (andyparsons) [reply config.toml:35]: reverting the other change and merging just this.

**Pull request merged**